packages/web-components/src/components/cta-section/cta-section.ts

```diff
@@ -64,6 +91,55 @@ class DDSCTASection extends StableSelectorMixin(DDSContentItem) {
     this[slotExistencePropertyNames[name] || '_hasFooter'] = hasContent;
   }
 
+  /**
+   * The observer for the resize of the content item slot
+   */
+  private _observeResizeContainer = () => {
+    const { _contentsNode: contentsNode } = this;
+
+    const childItems = contentsNode?.children[0].assignedNodes();
+    const newArray = new Array(childItems?.length);
+
+    childItems?.forEach((elem, index) => {
+      /* eslint-disable-next-line prefer-destructuring */
+      newArray[index] = elem.children[1];
+    });
+
+    newArray.forEach(entry => {
+      entry.style.height = 'auto';
+    });
+
+    if (window.innerWidth >= parseFloat(breakpoints.md.width) * baseFontSize) {
+      newArray.forEach(entry => {
+        entry.style.height = `${Math.max(
+          ...newArray.map(o => {
+            return o.clientHeight;
+          })
+        )}px`;
+      });
```

Comment:
Seems that the `clientHeight` calculation makes it <code>O(n<sup>2</sup>)</code> - Probably we should calculate the max height before setting `entry.style.height`?